examples/customresources/models/mygizmo/mygizmo.go

```diff
@@ -15,6 +15,22 @@ import (
 // Model is the full model definition.
 var Model = resource.NewModel("acme", "demo", "mygizmo")
 
+// Config is the gizmo model's config.
+type Config struct {
+	Arg1 string `json:"arg1"`
+}
+
+// Validate checks the attribute of myGizmo to ensure that an "arg1" attribute exists
+// the model will not initialize if this is not set
+func (cfg *Config) Validate(path string) ([]string, error) {
+	if cfg.Arg1 == "" {
+		return nil, fmt.Errorf(`expected "Arg1" attribute for myGizmo %q`, path)
```

Comment:
James - is there a reason myBase has the string wrapped in `` rather than ""?

Reply:
Backquotes are for raw string literals, and used because there are double-quotes inside the string itself (around the word "Arg1")